buildSrc/src/main/java/com/google/firebase/gradle/plugins/DackkaPlugin.kt

```diff
@@ -206,11 +206,20 @@ abstract class DackkaPlugin : Plugin<Project> {
             include("**/package-list")
         }.toList()
 
+    // TODO(b/243534168): Remove when fixed
+    private fun projectSpecificSources(project: Project) =
+        when (project.name) {
```

Comment:
Additionally I think you want to subtract `Timestamp.java` from firestore's source, otherwise it runs the risk of overwriting `com/google/firebase/package-summary.html` of firebase-common, Which I think is the case today in prod: https://firebase.google.com/docs/reference/kotlin/com/google/firebase/package-summary

Reply:
We do that in `projectSpecificSuppressedFiles`